lib/index.js

```diff
@@ -43,6 +43,33 @@ function initializer(key, base_url, options) {
   return this;
 }
 
+function parseResponse(err, response, body){
+  var data;
+  if (!body) {
+    data = err;
+  } else {
+    try {
+      data = JSON.parse(body);
+    } catch(e) {
+      data = body;
+    }
+  }
+
+  //request doesn't think 4xx is an error - we want an error for any non-2xx status codes
+  //we also we want this to be a real error object...
+  if (!err && (response.statusCode < 200 || response.statusCode > 206)){
```

Comment:
Why not check for `> 299` rather than ` > 206` to cover all 2xx?

Reply:
That's from the [original function](https://github.com/zencoder/zencoder-node/pull/5/files#diff-6d186b954a58d5bb740f73d84fe39073L85), so that one's on me, not @cgpuglie. It's been a while, but if I recall correctly this was something I saw in another SDK that took a similar approach to non-success code handling when using Request. I personally don't see any issue with just making the logic `> 299`, but to be fair 206 is the last ["official" 2xx status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).  ¯\\\_(ツ)_/¯